### PR TITLE
Create THOM CUP wiki page

### PR DESCRIPTION
**Pull request by WalterToro**
### Description

Created THOM CUP tournament wiki page

### Status

Finished

### CODE REVIEW

File: wiki/Tournaments/THOM_CUP/1/en.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+- Players are to ban one (1) map within the map pool that they are playing for the given round.
+
+- Players will be instructed to do a !roll command by the referee that is overseeing the match; the losing roll picks the first banned song, while the winning roll picks the first song of the match.
+
+- After the bans and rolls have concluded, the player with the winning roll instructs the referee of their song selection, and the referee will switch to that song. After that song concludes, the other player will pick a song, and this process will continue until one of the two conditions is satisfied:
+
+  1. One player has achieved enough wins in such a way that the other player cannot successfully win the round.
+
+    - The winning player moves onto the next match.
```
Comment (MegaApplePi) [inline wiki/Tournaments/THOM_CUP/1/en.md]: Is this supposed to be under the 1. item? If so, add another space:

```diff
-    - The winning player moves onto the next match.
+     - The winning player moves onto the next match.
```
This applies to the others below if so.

Comment (WalterToro) [reply wiki/Tournaments/THOM_CUP/1/en.md]: Seems to be okay on my end, I just tabbed twice in my editor and the preview looks fine, i dont know why it needs an extra space.

File: wiki/Tournaments/THOM_CUP/1/en.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+tags:
+- THOM
+- THOM CUP
+---
+THOM CUP #1
+======================
+
+![THOM CUP Logo](logo.png)
+
+## Preface
```
Comment (TPGPL) [inline wiki/Tournaments/THOM_CUP/1/en.md]: no reason to add this header i think

File: wiki/Tournaments/THOM_CUP/1/en.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+
+**THOM CUP** is a one-on-one euro-american tournament. As the title indicates, this tournament is for the osu!mania game mode. More specifically, this tournament is an osu!mania 7-key tournament.
```
Comment (TPGPL) [inline wiki/Tournaments/THOM_CUP/1/en.md]: **THOM CUP** is a one-on-one euro-american osu!mania tournament. It is the 1st installment of the THOM CUP.

may be better?

File: wiki/Tournaments/THOM_CUP/1/en.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+| --- | --- |
+| Host | ![][flag_CL] [WalterToro](https://osu.ppy.sh/users/5281416) |
+| Coordinator | ![][flag_AR] [juankristal](https://osu.ppy.sh/users/443656) |
+| Map Selectors | ![][flag_MY] [Cryolien](https://osu.ppy.sh/users/1626983), ![][flag_PL] [Kamikaze](https://osu.ppy.sh/users/2124783), ![][flag_US] [stupud man](https://osu.ppy.sh/users/2141612) |
+
+## Links
+
+- [Discussion Thread](/community/forums/posts/731431)
+- [Registration Form](https://docs.google.com/forms/d/e/1FAIpQLScuxNDPPjsI7-769vXteCdNDTzFcCJWa17XrRvyJvBfXa0pNQ/viewform)
+
```
Comment (TPGPL) [inline wiki/Tournaments/THOM_CUP/1/en.md]: add a horizontal line between those headers and add empty lines between it

File: wiki/Tournaments/THOM_CUP/1/en.md [hunk 5]
```diff
@@ -0,0 +1,192 @@
...
+- THOM CUP
+---
+THOM CUP #1
+======================
+
+![THOM CUP Logo](logo.png)
+
+## Preface
+
+**THOM CUP** is a one-on-one euro-american osu!mania tournament. It is the 1st installment of the THOM CUP. As the title indicates, this tournament is for the osu!mania game mode. More specifically, this tournament is an osu!mania 7-key tournament.
```
Comment (TPGPL) [inline wiki/Tournaments/THOM_CUP/1/en.md]: As the title indicates, this tournament is for the osu!mania game mode. 

this sentence is unneeded i think

File: wiki/Tournaments/THOM_CUP/1/en.md [hunk 6]
```diff
@@ -0,0 +1,190 @@
+---
+tags:
+- THOM
+- THOM CUP
+---
+THOM CUP #1
```
Comment (TPGPL) [inline wiki/Tournaments/THOM_CUP/1/en.md]: `#1 -> \#1`

File: wiki/Tournaments/THOM_CUP/1/en.md [hunk 7]
```diff
@@ -0,0 +1,190 @@
...
+======================
+
+![THOM CUP Logo](logo.png)
+
+**THOM CUP** is a one-on-one euro-american osu!mania tournament. It is the 1st installment of the THOM CUP. More specifically, this tournament is an osu!mania 7-key tournament.
```
Comment (TPGPL) [inline wiki/Tournaments/THOM_CUP/1/en.md]: `**THOM CUP** -> +**THOM CUP \#1**`

File: wiki/Tournaments/THOM_CUP/1/en.md [hunk 8]
```diff
@@ -0,0 +1,190 @@
...
+tags:
+- THOM
+- THOM CUP
+---
+THOM CUP \#1
+======================
+
+![THOM CUP Logo](logo.png)
+
+**THOM CUP \#1** (short for **"The Heptakey osu!mania Cup"**) is a one-on-one euro-american osu!mania tournament. It is the 1st installment of the THOM CUP. More specifically, this tournament is an osu!mania 7-key tournament.
```
Comment (TPGPL) [inline wiki/Tournaments/THOM_CUP/1/en.md]: `**THOM CUP \#1** (short for **"The Heptakey osu!mania Cup"**) ->  **The Heptakey osu!mania Cup \#1** (***THOM CUP \#1***)`

File: wiki/Tournaments/THOM_CUP/1/en.md [hunk 9]
```diff
@@ -0,0 +1,190 @@
+---
+tags:
+- THOM
+- THOM CUP
+---
+THOM CUP \#1
```
Comment (TPGPL) [inline wiki/Tournaments/THOM_CUP/1/en.md]: The Heptakey osu!mania Cup \#1

**Pull request merged**